Shared/Samples/Show line of sight between geoelements/ShowLineOfSightBetweenGeoelementsView.swift

```diff
@@ -0,0 +1,323 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowLineOfSightBetweenGeoelementsView: View {
+    /// The view model for the sample.
+    @State private var model = Model()
+    
+    /// A Boolean value indicating whether the settings sheet is presented.
+    @State private var isPresented = false
+    
+    var body: some View {
+        SceneView(
+            scene: model.scene,
+            graphicsOverlays: [model.graphicsOverlay],
+            analysisOverlays: [model.analysisOverlay]
+        )
+        .onAppear {
+            model.setupAnimation()
+        }
+        .onDisappear {
+            model.stopAnimating()
+        }
+        .overlay(alignment: .top) {
+            HStack {
+                Text("Visibility:")
+                Text(model.targetVisibility.label)
+            }
+            .frame(maxWidth: .infinity)
+            .padding(.vertical, 6)
+            .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                Button("Settings") {
+                    isPresented = true
+                }
+                .sheet(isPresented: $isPresented) {
+                    settingsSheet
+                }
+            }
+        }
+    }
+    
+    /// The settings configuration sheet for adjusting the observer's height.
+    private var settingsSheet: some View {
+        NavigationStack {
+            Form {
+                let heightRange = 20.0...70.0
+                LabeledContent(
+                    "Observer Height",
+                    value: Measurement(value: model.height, unit: UnitLength.meters),
+                    format: .measurement(width: .abbreviated)
+                )
+                Slider(
+                    value: $model.height,
+                    in: heightRange,
+                    step: 1
+                ) {
+                    Text("Observer Height")
+                } minimumValueLabel: {
+                    Text(Measurement(value: heightRange.lowerBound, unit: UnitLength.meters).formatted())
+                } maximumValueLabel: {
+                    Text(Measurement(value: heightRange.upperBound, unit: UnitLength.meters).formatted())
```

Comment:
These can also use the Text [format](https://developer.apple.com/documentation/swiftui/text/init(_:format:))